app/models/in_person_enrollment.rb

```diff
@@ -151,6 +151,10 @@ def usps_unique_id
     user.uuid.delete('-').slice(0, 18)
   end
 
+  def enhanced_ipp?
```

Comment:
Heads-up, I'm going to use this logic in my current PR, too.  Thanks for adding it. 👍🏻